### PR TITLE
Fix character power bars in armory

### PR DESCRIPTION
**Pull request by CraftedRO**
-This will fix the power bars for respective classes in armory : 
- Mana / Focus for Hunters
- Energie for Rogues
- Rage for Warriors
- Runic Power for Death Knights.
--And yes this is commit worthy for srp6 too.

Captureqw

### CODE REVIEW

**Comment by favorit5503** [comment]
My question is. What was the problem before the fix?

**Comment by OMGhixD-OG** [comment]
If i recall correctly we only had RAGE/ENERGY? And this adds the remaining ones?

File: application/modules/character/controllers/character.php [hunk 1]
```diff
@@ -170,35 +170,45 @@ private function getInfo()
...
+            switch($this->className)
+            {
+                default:
+                    $this->secondBar = "mana";
+                    $this->secondBarValue = $this->stats['maxpower1'];
+                break;
+
+                case "Warrior":
+                    $this->secondBar = "rage";
+                    $this->secondBarValue = $this->stats['maxpower2'] / 10;
```
Comment (OMGhixD-OG) [inline application/modules/character/controllers/character.php:183]: Why "/ 10"?

File: application/modules/character/controllers/character.php [hunk 2]
```diff
@@ -170,35 +170,45 @@ private function getInfo()
...
+                break;
+
+                case "Rogue":
+                    $this->secondBar = "energy";
+                    $this->secondBarValue = $this->stats['maxpower4'];
+                break;
+
+                case "Hunter":
```
Comment (OMGhixD-OG) [inline application/modules/character/controllers/character.php:191]: Genuinly a nice fix! Well done

**Comment by OMGhixD-OG** [comment]
In its entirety i feel like its a bit useless to have maxpower showcasement on armory/character-view as the only thing worthy of showcasing would be Mana. Everything else is either 100 or can only become 100.

But that is something we'll have to discuss. I do approve of this tho :)

**Review by OMGhixD-OG — APPROVED** [review]
This is a good and safe change.

It ads post/pre cata support changes for Hunters and this is a rather nice change we need/want.

But as a over-all note: I personally do not see a reason to include Runic, Energy, Focus or Rage as they can only reach value 100 or 0. The only thing a user would have usefulness of seeing would be mana as that is the only maxpower that would reach a significant value (or unregular value)

Regardless, i do approve of this PR. Well done!

**Comment by CraftedRO** [comment]
well they can get over 100 if u use respective talents of class , dk : runic power mastery 2/2 :

Captureza

**Comment by OMGhixD-OG** [comment]
> well they can get over 100 if u use respective talents of class , dk : runic power mastery 2/2 :
> 
> Captureza

Ohh yeah i forgot. Nonetheless tho, it will never reach a significant value to where it'll play any "specific" role or matter on the armory/character view. Most people use armories etc etc to follow metas and whatnot.

**Pull request merged**